.semaphore/README.md

```diff
@@ -0,0 +1,48 @@
+# Semaphore CI 2.0
```

Comment:
Let's remove Semaphore set up stuff as we will be using Github Actions for this project

Reply:
Removed in https://github.com/nimblehq/nimble-survey-web/pull/1/commits/86e0fb91c47254aab0053bcb04614daa90bd521b